compliance/collection/nodeinventorizer/nodeinventory.go

```diff
@@ -0,0 +1,77 @@
+package nodeinventorizer
+
+import (
+	timestamp "github.com/gogo/protobuf/types"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/scanner/database"
+	scannerV1 "github.com/stackrox/scanner/generated/scanner/api/v1"
+	"github.com/stackrox/scanner/pkg/analyzer/nodes"
+)
+
+// NodeInventorizer is the interface that defines the interface a scanner must implement
+type NodeInventorizer interface {
```

Comment:
How come we changed the name from `NodeScanner`? I figure if we define a `Scan` function, `NodeScanner` makes sense. I don't think `Inventorizer` is a word, so I'm a bit skeptical using it to name this interface